etc/notebooks/meetup_outreach_demo/meetup-outreach.ipynb

```diff
@@ -1383,157 +1383,159 @@
     "        'use strict';\n",
     "\n",
     "        HTMLImports.whenReady(function() {\n",
-    "            Polymer({\n",
-    "                is: 'dynamic-list',\n",
-    "\n",
-    "                properties: {\n",
-    "                    heading: {\n",
-    "                        type: String,\n",
-    "                        value: 'Dynamic List'\n",
-    "                    },\n",
-    "                    items: {\n",
-    "                        type: Array,\n",
-    "                        value: function() { return []; }\n",
+    "            if(!window.dynamicList) {\n",
```

Comment:
Since this isn't a standard thing to do in Polymer, you should add a comment about why we're saving to a global and checking -- that this is all about re-running this cell within the Notebook/Dashboard environment.